src/nix/profile.md

```diff
@@ -12,7 +12,7 @@ them to be rolled back easily.
 The default profile used by `nix profile` is `$HOME/.nix-profile`,
 which, if it does not exist, is created as a symlink to
 `/nix/var/nix/profiles/default` if Nix is invoked by the
-`root` user, or `/nix/var/nix/profiles/per-user/`*username* otherwise.
+`root` user, or `${XDG_STATE_HOME-$HOME/.local/state}/nix/profiles/profile` otherwise.
```

Comment:
I actually think we shouldn't document that location at all. This shouldn't matter to the user and keeping it opaque would probably be better for everyone